packages/web-components/src/components/cta-section/cta-section.ts

```diff
@@ -23,6 +23,8 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
  * The CTA Section pattern
  *
  * @element dds-cta-section
+ * @slot heading - The text heading.
+ * @slot buttons - The CTA Buttons.
  */
 @customElement(`${ddsPrefix}-cta-section`)
 class DDSCTASection extends StableSelectorMixin(LitElement) {
```

Comment:
Seems that this class should inherit `DDSContentItem`.

Reply:
Thanks for the heads up! I'm wondering though, what are the main reasons it should inherit `DDSContentItem`?

Reply:
Thank you for asking @IgnacioBecerra! One reason is that `<dds-cta-section>` seems to do a similar thing to `<dds-content-item>`, especially from the perspective of the API set. Another reason is that `<CTASection>` in React renders `<ContentItem>`, effectively extending `<ContentItem>` via [React composition pattern](https://reactjs.org/docs/composition-vs-inheritance.html), whereas React composition pattern doesn't work well with Web Components: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/00ad9c7/packages/web-components/docs/coding-conventions.md#preferring-class-inheritance-pattern-over-react-composition-pattern